spec/unit/policyfile_includes_dsl_spec.rb

```diff
@@ -57,6 +57,67 @@
       end
     end
 
+    describe "and the included policy and policy_revision_id are correctly configured" do
```

Comment:
I'd like to see some coverage of handling when revisions don't match. Again I don't think I'd block on this but it'd give me a little more confidence.

Reply:
Given the spec/unit/policyfile_includes_dsl_spec.rb is really just testing the interface, not the implementation I'm open to suggestions for how to test it. I'm not the best at RSpec.